packages/collabswarm/src/auth-subtlecrypto.ts

```diff
@@ -71,11 +71,10 @@ export class SubtleCrypto
   _encryptionAlgorithmParams(nonce?: Uint8Array): AesGcmParams {
     switch (this._encryptionAlgorithmName) {
```

Comment:
Do we just want to remove `_encryptionAlgorithmName` since it's not used anymore?

Reply:
Isn't this called still in both `decrypt` and `encrypt`?

Reply:
They call this `_encryptionAlgorithmParams` function, not the `_encryptionAlgorithmName` directly